servicemesh-release-notes/topics/rn-known-issues.adoc

```diff
@@ -71,16 +71,14 @@ pass:quotes[*kubeConfigFile: /dev/null*]
 
 Create a soft link from `/var/lib/origin/null` to `/dev/null` and replace the two kubeConfigFile variables to `/var/lib/origin/null`. This makes the task think /var/lib/origin/null is an asset in the origin "perimiter" and continue running.
 
-https://issues.jboss.org/browse/MAISTRA-62[MAISTRA-62] After a successful Istio installation, when upgrading an OCP cluster from 3.10 to 3.11, the storage upgrade task fails causing the entire upgrade process to fail.
+https://issues.jboss.org/browse/MAISTRA-245[MAISTRA-245] The sidecar injector pod fails to start if you are running the upstream community version.
```

Comment:
It looks like there is a workaround for this issue that should also be documented.